dojo/fixtures/test_type.json

```diff
@@ -492,14 +492,20 @@
     	},
     	"model": "dojo.test_type",
     	"pk": 162
-
 	},
 	{
 		"fields": {
-      		"name": "Aqua Scan"
-    	},
-    	"model": "dojo.test_type",
-    	"pk": 163
+			"name": "Aqua Scan"
+		},
+		"model": "dojo.test_type",
+		"pk": 163
+	},
+	{
+		"fields": {
+			"name": "Dependency Track Finding Packaging Format (FPF) Export"
+		},
+		"model": "dojo.test_type",
+		"pk": 164
 
-	}
+	},
```

Comment:
@Maffooch thanks for fixing my botched attempt at resolving the merge conflict. Every time I tried to pull or push it said everything was up to date so I am not sure what was going on. There is a trailing comma here that might cause some issues though!

Reply:
You're welcome! It can be a bit wonky sometimes. The trailing comma can be there or not in this case. May as well save the next person a keystroke :)